tests/integration/synapseclient/test_evaluations.py

```diff
@@ -110,13 +110,13 @@ def test_evaluations():
         for submission in submissions:
             assert_true(re.match('Submission \\d+', submission['name']))
             status = syn.getSubmissionStatus(submission)
-            status.score = random.random()
+            # status.score = random.random()
             if submission['name'] == 'Submission 01':
                 status.status = 'INVALID'
-                status.report = 'Uh-oh, something went wrong!'
+            #    status.report = 'Uh-oh, something went wrong!'
             else:
                 status.status = 'SCORED'
-                status.report = 'a fabulous effort!'
+            #    status.report = 'a fabulous effort!'
```

Comment:
Are these fields and above no longer used?

Reply:
Ya, `status.report` is now deprecated